src/image_selector.py

```diff
@@ -43,13 +43,14 @@ def post_now(filename, mode):
     account_copy = select_account(account_data, scheduler_profile_ids=scheduler_profile_ids)
     account_copy.set_config_for(filename)
     caption = ImageMetadataAdjuster(filename).get_caption() or ""
+    content_tags = ImageMetadataAdjuster(filename).get_content_tags() or ""
 
     response = False
     if mode == "Twitter":
-        response = TwitterClient(account_copy).schedule(filename, caption)
+        response = TwitterClient(account_copy).schedule(filename, caption, content_tags)
```

Comment:
will this work? twitter doesn't have this arg assigned anymore

Reply:
Ah yes, missed that, thx.